scipy/stats/_qmc.pyi

```diff
@@ -1,138 +0,0 @@
-# Prevents annotations from being evaluated during runtime,
-# thus storing them as plain strings
-from __future__ import annotations
-
-from abc import ABC, abstractmethod
-from typing import (
-    TYPE_CHECKING,
-    Optional,
-    Union,
-    type_check_only,
-)
-
-import numpy as np
-
-if TYPE_CHECKING:
-    import numpy.typing as npt
```

Comment:
This is needed for NumPy < 1.20

Reply:
It is not used in [`SciPy/stats/_entropy.py`](https://github.com/scipy/scipy/blob/master/scipy/stats/_entropy.py) where there are also type annotations

Reply:
Sure because it uses directly in the type hints `np.typing` then this is not evaluated at runtime. But here you import `numpy.typing` so this will only work for NumPy > 1.2. Hence the failing jobs.

Reply:
Ok will try this solution then